### PR TITLE
Add multiple option to yargs

### LINKED ISSUE #207: Add yargs option to support importing from multiple linked tsfiles
https://github.com/fable-compiler/ts2fable-exports/commit/8f6b6148cdd9d1bb24a07164e6143788f3c457db
`office-ui-fabric-react` generated an almost empty file

I want to embed `multiple` option to `yargs` .
It has description `Import from multiple linked tsfiles`
Arguments are a entry `tsfile` and a output `fsfile`

This is a beginning to get `office-ui-fabric-react` worked

For example:
`ts2fable --multiple node_modules/office-ui-fabric-react/lib/index.d.ts
test-compile/OfficeUiFabricReact.fs`
imgr

### PR DESCRIPTION
**Pull request by humhei**
fix #207

### CODE REVIEW

File: src/ts2fable.fs [hunk 1]
```diff
@@ -11,28 +11,39 @@ open ts2fable.Naming
...
+        failwithf "TypeScript file not found: %s" ts
 
+match tsfiles.Length, fsfile,isMultiple with
+| 0, _, _ -> failwithf "Please provide the path to a TypeScript file"
+| _, None, _ -> failwithf "Please provide the path to the F# file to be written "
+| _, Some fsf, false ->
+    printfn "ts2fable %s" Version.version
     writeFile tsfiles fsf
+| 1, Some _, true ->
+    printf "Not implemented"
```
Comment (humhei) [inline src/ts2fable.fs]: `Not implemented`
This code will be replaced ：
Resolving entry tsfile to multiple input tsfiles using `Typescript Compile API`

File: build.fsx [hunk 2]
```diff
@@ -239,6 +243,10 @@ Target.Create "WatchTest" (fun _ ->
     runDotnet toolDir "fable webpack -- --config webpack.config.test.js -w"
 )
 
+Target.Create "WatchCli" (fun _ ->
+    runDotnet toolDir "fable webpack -- --config webpack.config.cli.js -w"
```
Comment (humhei) [inline build.fsx]: WachCli is used to debug yargs

File: build.fsx [hunk 3]
```diff
@@ -89,36 +89,40 @@ Target.Create "BuildCli" (fun _ ->
...
             ]
-            "test-compile/Protobuf.fs"
-        ts2fable ["node_modules/synctasks/dist/SyncTasks.d.ts"] "test-compile/SyncTasks.fs"            
-        ts2fable ["node_modules/subscribableevent/dist/SubscribableEvent.d.ts"] "test-compile/SubscribableEvent.fs"            
-        ts2fable ["node_modules/office-ui-fabric-react/lib/index.d.ts"] "test-compile/OfficeUiFabricReact.fs"            
+        ts2fable ["node_modules/synctasks/dist/SyncTasks.d.ts";"test-compile/SyncTasks.fs"]           
+        ts2fable ["node_modules/subscribableevent/dist/SubscribableEvent.d.ts";"test-compile/SubscribableEvent.fs"]           
+        ts2fable 
+            [
+                "--multiple"
```
Comment (humhei) [inline build.fsx]: `--multiple` Sample

**Comment by humhei** [comment]
> We need to discuss more

I totally agree! 👍

**Comment by ctaggart** [comment]
Replaced by #216. Please review it

**Pull request closed without merging**